### DISCUSSION TITLE
What additional steps to take to secure WAN exposed https proxied frigate service to authenticated port 8971 ?

### DISCUSSION THREAD

**Discussion by chrislawso** [blakeblackshear/frigate#13919] [Ask A Question]
Without using vpn or tunnels, only using a separate reverse proxy for 443 to frigate 8971, what more can be performed to secure frigate that is exposed to internet? Side note, I do not want to maintain and actively manage ip whitelists on firewall of allowed remote users.

For example I do not see fail2ban integrated with frigate and I uncertain what would be required to add fail2ban to the frigate stable docker compose environment.

Anyone doing any other additional security measures?

**Comment by NickM-27** [comment]
can't really help since I use cloudflare. I use Frigate's authentication but use cloudflare to limit the scope of access to certain countries, remove known problematic IPs, etc.

**Comment by sawokei** [comment]
did you find a possibility to integrate fail2ban or even crowdsec into a docker installation?
